core/trino-spi/src/main/java/io/trino/spi/predicate/SortedRangeSet.java

```diff
@@ -1151,11 +1166,30 @@ public String toString()
 
         public String formatRange(ConnectorSession session)
         {
+            Object lowValue;
+            if (isLowUnbounded()) {
+                lowValue = "min";
```

Comment:
is "min" more readable than "null" for denoting lack of low bound?

Reply:
At first glance I was confused on seeing DF stats like `[ SortedRangeSet[type=bigint, ranges=1, {(null,null)}] ]`.
I was used to seeing min/max from the previous code in Range.
We are using `<min>` and `<max>` `PlanPrinter#formatDomain` as well.
I'm ok to settle on some common sentinel values in all places.